app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -62,100 +62,85 @@ You’ll earn a [competitive salary](/salaries-and-benefits), get supported in y
+You will not have to pay a fee or undergo further training. However, to be awarded QTS you will need to meet certain criteria. [Learn more about the qualifications and experience you'll need to get QTS](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers). 
 
 <p class="call-to-action__action">
   <a href="https://apply-for-qts-in-england.education.gov.uk/eligibility/start">Apply for QTS <span>in England</span></a>
 </p>
 
 
-### Other routes to QTS
-
-If you do not meet all of these requirements you may still be able to:
+If you’re not eligible to use the new service, you may be able to:
 
-* apply for QTS through the assessment only route
-* complete a teacher training course in England leading to QTS
+* [complete initial teacher training in England](https://getintoteaching.education.gov.uk/non-uk-teachers/train-to-teach-in-england-as-an-international-student)
+* [work without QTS and then undergo the assessment only route](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk#apply-for-assessment-only-qts)
+* [apply for international qualified teacher status (iQTS)](https://getintoteaching.education.gov.uk/non-uk-teachers/international-qualified-teacher-status)
 
-[Learn more about routes to QTS for teachers who trained or have teaching experience outside the UK](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk#qts-exemption-for-teachers-from-outside-the-uk).
 
-## Search for a job
+## Find a teaching job
 
-To find a teaching job, you must apply to individual schools (not the Department for Education). 
+Once you've obtained QTS, you will need to find a teaching job in England. To do this, you must apply to individual schools (not the Department for Education). 
 
-You can search for teaching jobs by region, subject specialism and age group.
+You can search and apply for jobs by region, subject specialism and age group using [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/?gclid=Cj0KCQiA_P6dBhD1ARIsAAGI7HCw9dQ-qerRqmVac1uNempXRIiJOdaSkJDNUjpMmBU4QsnKbYzn8ogaAvapEALw_wcB&gclsrc=aw.ds).
 
 Most teaching vacancies are advertised in March, April or the first half of May, with jobs usually starting at the beginning of the school year in September. 
 
+Select a job to apply for, and contact the school to check they are open to sponsoring skilled worker visas. Submit your application, and if you are offered the job, you’ll then be able to apply for your visa.
+
+
 Set up a Teaching Vacancies job alert to be updated about opportunities throughout the year.
```

Comment:
Suggestion:
Once you’ve obtained QTS, you will need to find a teaching job in England. 

You can search and apply for jobs by region, subject specialism and age group.

Most teaching vacancies are advertised in March, April or the first half of May, with jobs usually starting at the beginning of the school year in September.

When you've found a job to apply for, you should contact the school to check they are open to sponsoring skilled worker visas. You can then submit your application directly to the school (not the Department for Education), and if you are offered the job, you’ll then be able to apply for your visa.

You can also set up alerts to be updated about opportunities as they become available.

Button text: Search for teaching jobs in England